.github/workflows/serverless-vuln-scan.yml

```diff
@@ -0,0 +1,75 @@
+name: "Serverless Vulnerability Scan"
+
+on:
+  schedule:
+    # daily at midnight
+    - cron: "0 0 * * *"
+
+env:
+  VERSION: 1  # env var required when building extension
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout datadog-agent repository
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
Why are we adding hashes instead of the version?

Reply:
I copied this from the agent repo. It was changed to hashes for security reasons.